docs/LV_client/tutorials/wheel_integration.md

```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
+## Pre-requisites
+1. The **mono_closed_loop_example_wheel.vi** and the **mono_replay_example_wheel.vi** use the **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).
+1. Install the Logitech G HUB to obtain the latest drivers for the wheel from [here](https://www.logitechg.com/en-us/innovation/g-hub.html)
+1. Download the Logitech Steering Wheel SDK from [here](https://www.logitechg.com/en-us/innovation/developer-lab.html)
+1. Create a LabVIEW library using the **LogitechSteeringWheelEnginesWrapper.dll** using the Import Shared Library Wizard, you can find a tutorial on how to do this [here](http://zone.ni.com/reference/en-XX/help/371361R-01/lvhowto/example_import_shared_library/).
+1. Install the **Maker Hub Interface for Xbox One Controller** using the VIPM.
+
+## How to run
+1. Connect the wheel to the computer. 
+1. Open the **mono_closed_loop_example_wheel.vi** or the **mono_replay_example_wheel.vi** 
+1. Run the selected VI.
+
+## Closed Loop with Wheel 
+<div class="img_container">
+  <video width=650px height=440px muted controls autoplay loop>
+    <source src="http://cdn.monodrive.io/readthedocs/closed-loop.mp4" type="video/mp4">
```

Comment:
This is amazing. The monitors are a little off because one is higher in the scene than the other. But otherwise, looks great!!! This is such a neat tool.